### PR TITLE
T365569: release 3.0.0 (WBS Deploy 3.0.0)

### PR DESCRIPTION
**Pull request by rti**
**⛔️ Merging this PR will make the release to DockerHub. Don't merge until the [release announcement](https://phabricator.wikimedia.org/T366067) is complete and sent.**

---

#### WBS Images

| Image               | WBS Version | Upstream Version     | Base Image                    | Distribution    |
| ------------------- | ----------- | -------------------- | ----------------------------- | --------------- |
| WBS Wikibase        | 3.0.0       | MediaWiki 1.42.1     | PHP 8.3.8 apache              | Debian Bookworm |
| WBS WDQS            | 1.0.0       | WDQS 0.3.142         | JRE Eclipse Temurin 8u412-b08 | Debian Bookworm |
| WBS WDQS Frontend   | 1.0.0       | N/A                  | Nginx 1.27.0                  | Debian Bookworm |
| WBS WDQS Proxy      | 1.0.0       | N/A                  | Nginx 1.27.0                  | Debian Bookworm |
| WBS QuickStatements | 1.0.0       | N/A                  | PHP 8.3.8 Apache              | Debian Bookworm |
| WBS ElasticSearch   | 1.0.0       | ElasticSearch 7.10.2 | N/A                           | N/A             |

#### WBS Deploy 3.0.0

https://phabricator.wikimedia.org/T365569

On top of the images above, WBS Deploy uses:

| Image               | Upstream Version     |
| ------------------- | -------------------- |
| MariaDB             | 10.11                |
| Traefik             | 3.1                  |

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -245,3 +245,4 @@ done
 if ! $build_target_set; then
     build_all
 fi
+
```
Comment (rti) [inline build.sh:248]: just here to trigger the pipeline

**Review by lorenjohnson — APPROVED** [review]
This can go ahead when we're ready to release AND the `T365569-release-deploy-3.0.0` is current with `main`.

**Review by lorenjohnson — APPROVED** [review]
Looks up to date with main. Good to merge.

**Review by darthmon — APPROVED** [review]
Changes look good.
Looked up the commit hashes and they check.
Tested (all) locally and it works.
I am running it locally at the moment and it all works fine.

**Pull request merged**